editor/filesystem_dock.cpp

```diff
@@ -2152,6 +2152,11 @@ void FileSystemDock::_file_list_rmb_pressed(const Vector2 &p_pos) {
 	file_list_popup->popup();
 }
 
+void FileSystemDock::_script_created(Ref<Script> /* p_script */) {
```

Comment:
If the method takes an argument, it should be named (even if unused), not commented out